package.json

```diff
@@ -11,8 +11,8 @@
     "@storybook/storybook-deployer": "^2.0.0",
     "axios-mock-adapter": "^1.10.0",
     "babel-cli": "^6.10.1",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^6.1.2",
+    "babel-core": "^6.26.0",
```

Comment:
why are you downgrading ?

Reply:
Nice catch! I think it because of some rebase.

Reply:
Fixed